server/node-postgres-mydb/db/seeds/dev/04_artist.js

```diff
@@ -0,0 +1,18 @@
+const songs = require(' ../../../seed.json');
+
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex('artist').del()
+    .then(function () {
+      const artistsList =[];
+    artistsList.map(artist=> {
```

Comment:
Same problem as in `03_band_has_artist.js`:

You iterate over an empty array `artistsList` and you don't actually pull any data from the json file.

Reply:
yes I got it my mistake in both these tables I mapping over empty record